nemo/collections/tts/helpers/helpers.py

```diff
@@ -372,3 +372,37 @@ def eval_tts_scores(
     ## fs was set 16,000, as pesq lib doesnt currently support felxible fs.
 
     return {'STOI': stoi_score, 'PESQ': pesq_score}
+
+
+def regulate_len(durations, enc_out, pace=1.0, mel_max_len=None):
```

Comment:
Do we need the FastPitch license added to the header of this file w/ this function moved?

Reply:
done